src/accessibility/RovingTabIndex.tsx

```diff
@@ -308,7 +308,7 @@ export const useRovingTabIndex = <T extends HTMLElement>(
     inputRef?: RefObject<T>,
 ): [FocusHandler, boolean, RefObject<T>] => {
     const context = useContext(RovingTabIndexContext);
-    let ref = useRef<T>(null);
+    let ref: RefObject<T> = useRef<T>(null);
```

Comment:
Is this necessary? Playing on the branch it seems to me that TS correctly infers this type (exactly as you've annotated here).